xilem_core/src/view.rs

```diff
@@ -208,6 +208,9 @@ where
 #[allow(unnameable_types)] // reason: Implementation detail, public because of trait visibility rules
 pub struct RcState<ViewState> {
     view_state: ViewState,
+    /// This is a flag that is set, when an inner view signifies that it requires a rebuild (via [`MessageResult::RequestRebuild`]).
+    /// This can happen, e.g. when an inner view wasn't changed by the user directly, but e.g. some kind of async action.
```

Comment:
It's not really the user, it's effectively the app developer, right?

I guess the semantics of who is the user isn't that clear - is it the end user, or the user of Xilem?